src/Filter/StringFilter.php

```diff
@@ -52,14 +56,20 @@ public function filter(ProxyQueryInterface $queryBuilder, $alias, $field, $data)
         if (ChoiceType::TYPE_EQUAL == $data['type']) {
             $queryBuilder->setParameter($parameterName, $data['value']);
         } else {
-            $queryBuilder->setParameter($parameterName, sprintf($this->getOption('format'), $data['value']));
+            $queryBuilder->setParameter($parameterName,
+                sprintf(
+                    $this->getOption('format'),
+                    $this->getOption('case_sensitive') ? $data['value'] : mb_strtolower($data['value'])
```

Comment:
what is the point of this? why do you need `case_sensitive` as a query parameter?

Reply:
no need for query parameter, but for the value which will then be use in the query:

before `%FooBar%` -> after `%foobar%`

Reply:
I updated the test

Reply:
oooooo, I totally ignored the ternary :sweat_smile: